package.json

```diff
@@ -6,8 +6,8 @@
   "main": "dist/node-cjs.js",
   "module": "dist/node-esm.js",
   "browser": {
-    "dist/node-cjs.js": "dist/browser-cjs.js",
-    "dist/node-esm.js": "dist/browser-esm.js",
+    "dist/node-cjs.js": "./dist/browser-cjs.js",
+    "dist/node-esm.js": "./dist/browser-esm.js",
     "stream": "stream-browserify"
   },
   "types": "dist/index.browser.d.ts",
```

Comment:
Wouldn't this need to be changed as well then?

Reply:
Yeah, that should also be updated, but it looks like this PR will soon be OBE due to PR #680. So I may just hold out for that and close this PR.